client/app/components/SettingsWrapper.less

```diff
@@ -0,0 +1,17 @@
+@import '~antd/lib/menu/style/index';
+
+.settings-screen {
+  .@{menu-prefix-cls} {
```

Comment:
I don't think this should be scoped to this page only. Our tabs should have the same look everywhere (unless it makes sense to have a different look in a specific component).

Btw, I said it needs to be consistent, but I didn't say to which direction :-) We can also make the vertical tabs consistent with the horizontal tabs. I think right now they still use bootstrap classes/components, we can try and switch them to Ant component.

Reply:
I like variant 2 :slightly_smiling_face:

Reply:
You mean migrate the vertical tabs? Me too :)

Then let's revert the color change here and have a separate task of migrating the vertical tabs? It's a single place for all the list pages, right?

Reply:
I went with "this page only" approach because it was the valid one in this scope, but let's discuss the second option in #4338 :)